swarm/api/routes/db.py

```
@@ -283,6 +296,17 @@ async def ingest_run_events(run_id: str):
     Returns:
         Dict with ingestion statistics.
     """
+    try:
+        validate_path_component(run_id, "run_id")
+    except ValueError as e:
+        return {
+            "run_id": run_id,
+            "success": False,
+            "events_ingested": 0,
+            "error": str(e),
+            "timestamp": datetime.now(timezone.utc).isoformat(),
```

Ruff (0.14.14)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)